### PR TITLE
Remove remaining basic uses of @_ver in git_hashtag

### PR DESCRIPTION
**Pull request by Zopolis4**
Following on from #8374, I am removing 44 uses of `@_ver` from packages that use it like this:
```
@_ver = '1.2.3'
version @_ver
...
git_hashtag @_ver
```
which now becomes:
```
version '1.2.3'
...
git_hashtag version
```

Tested and working on `x86_64`.

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/Zopolis4/chromebrew.git CREW_TESTING_BRANCH=laver4 CREW_TESTING=1 crew update
```

### CODE REVIEW

**Comment by uberhacker** [comment]
As @supechicken mentioned earlier, I don't think we want to replace all instances of `@_ver` globally since in several instances, partial data is extracted from the version sometimes.

File: packages/evolution_data_server.rb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 class Evolution_data_server < Package
   description 'Centralized access to appointments and contacts'
-  @_ver = '3.48.1'
   version @_ver
```
Comment (uberhacker) [inline packages/evolution_data_server.rb:5]: `version '3.48.1'`?

File: packages/harfbuzz.rb [hunk 2]
```diff
@@ -4,7 +4,6 @@
 class Harfbuzz < Package
   description 'HarfBuzz is an OpenType text shaping engine.'
   homepage 'https://www.freedesktop.org/wiki/Software/HarfBuzz/'
-  @_ver = '7.1.0'
   version @_ver
```
Comment (uberhacker) [inline packages/harfbuzz.rb:7]: `version '7.1.0'`?

File: packages/libgsf.rb [hunk 3]
```diff
@@ -3,8 +3,7 @@
 class Libgsf < Package
   description 'The G Structured File Library'
   homepage 'https://gitlab.gnome.org/GNOME/libgsf'
-  @_ver = '1.14.50'
-  @_ver_prelastdot = @_ver.rpartition('.')[0]
+  '1.14.50'_prelastdot = @_ver.rpartition('.')[0]
```
Comment (uberhacker) [inline packages/libgsf.rb:6]: Definitely don't want to do this.

File: packages/libproxy.rb [hunk 4]
```diff
@@ -3,12 +3,11 @@
 class Libproxy < Package
   description 'libproxy is a library that provides automatic proxy configuration management.'
   homepage 'https://libproxy.github.io/libproxy/'
-  @_ver = '5d5e13ddc47a2a061c595c1356d7d07d78cf597f'
-  version @_ver[0, 7]
+  version '5d5e13ddc47a2a061c595c1356d7d07d78cf597f'[0, 7]
```
Comment (uberhacker) [inline packages/libproxy.rb:6]: `version '5d5e13d'`?

File: packages/opam.rb [hunk 5]
```diff
@@ -6,7 +6,6 @@
 class Opam < Package
   description 'OCaml package manager'
   homepage 'https://opam.ocaml.org/'
-  @_ver = '2.1.4'
   version @_ver
```
Comment (uberhacker) [inline packages/opam.rb:9]: `version '2.1.4'`?

File: packages/pango.rb [hunk 6]
```diff
@@ -4,7 +4,6 @@
 class Pango < Package
   description 'Pango is a library for laying out and rendering of text, with an emphasis on internationalization.'
   homepage 'https://pango.gnome.org/'
-  @_ver = '1.50.14'
   version @_ver
```
Comment (uberhacker) [inline packages/pango.rb:7]: `version '1.50.14'`?

File: packages/pipewire.rb [hunk 7]
```diff
@@ -3,7 +3,6 @@
 class Pipewire < Package
   description 'PipeWire is a project that aims to greatly improve handling of audio and video under Linux.'
   homepage 'https://pipewire.org'
-  @_ver = if Gem::Version.new(CREW_KERNEL_VERSION.to_s) < Gem::Version.new('3.9')
```
Comment (uberhacker) [inline packages/pipewire.rb:6]: I don't think you will want to remove this line only.

**Comment by Zopolis4** [comment]
Oops-- I rushed this PR out due to time pressure, evidently I made a few mistakes. Will fix it when I've got the time.

**Comment by uberhacker** [comment]
You're supposed to check (and hopefully test) your updates prior to submitting them. :wink:

**Comment by Zopolis4** [comment]
Yeah, lesson learned :disappointed_relieved:

At any rate, fixed the sed script to make less assumptions:
```
#!/bin/bash

for package in *.rb; do
  if grep -q "version @_ver" "${package}"; then
    if [[ $(grep -c "@_ver" "${package}") == "3" ]]; then
      if grep -q "git_hashtag" "${package}"; then
        if grep -q 'git_hashtag @_ver' "${package}"; then
          if grep -Fq "@_ver = '" "${package}"; then
            avln=$(grep -Fhn "@_ver = '" "${package}" | cut -f1 -d:)
            vversion=$(sed -n "${avln}s/@_ver = \(.*\)/\1/;${avln}s/^[ \t]*//;${avln}s/[ \t]*$//p" "${package}" | tr -d "'")
            sed -i "${avln}d" "${package}"

            vavln=$(grep -Fhn "version @_ver" "${package}" | cut -f1 -d:)
            sed -i "${vavln}s/@_ver/'$vversion'/" "${package}"

            ghavln=$(grep -Fhn "git_hashtag @_ver" "${package}" | cut -f1 -d:)
            sed -i "${ghavln}s/@_ver/version/" "${package}"
          fi
        fi
      fi
    fi
  fi
fi
```

when it comes to `sed`, coherence is the enemy of functionality.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**